spacy/pipeline/entity_linker.py

```diff
@@ -292,6 +345,8 @@ def batch_has_learnable_example(self, examples):
 
         for eg in examples:
             for ent in eg.predicted.ents:
+                # todo Do we want batching here too? I'm assuming that in most cases this finds a learnable example soon
+                #  anyway - and stops afterwards - so fetching candidates for the entire batch is maybe not faster.
```

Comment:
See comment in code. I think batching here might not be that useful, but I'm also not sure about the performance implications of `has_learnable_example()`.

Reply:
Ah difficult to say. Maybe we can leave this as is for now.